apps/asap-server/src/utils/types.ts

```diff
@@ -38,3 +38,8 @@ export type ScheduledHandlerAsync = (
 export type DeepWriteable<T> = {
   -readonly [P in keyof T]: DeepWriteable<T[P]>;
 };
+
+declare global {
+  type Instant = string & { __type: 'SquidexDate' };
```

Comment:
these are Squides scalars, `Instant` is the date one, i was tempted to rename (if its possible at all) but I guess im gonna drop it for now.

The `& {__type: '....'}` is a way to create a scalar which is not a simple string that you could just map over (like we dont want to allow any string to match our Instant type)